src/pages/patch/patchTabs/Tasks.tsx

```diff
@@ -46,45 +46,36 @@ export const Tasks: React.FC<Props> = ({ taskCount }) => {
 
   const { search } = useLocation();
 
-  const [initialQueryVariables] = useState(
-    getQueryVariables(search, resourceId)
-  );
+  const queryVariables = getQueryVariables(search, resourceId);
 
-  const { sortBy, sortDir } = initialQueryVariables;
+  const { sortBy, sortDir, limit, page } = queryVariables;
 
   const columns = useSetColumnDefaultSortOrder<TaskResult>(
     columnsTemplate,
     sortBy,
     sortDir
   );
 
-  const { data, error, networkStatus, refetch } = useQuery<
+  const { data, error, loading, startPolling, stopPolling } = useQuery<
     PatchTasksQuery,
     PatchTasksQueryVariables
   >(GET_PATCH_TASKS, {
-    variables: initialQueryVariables as PatchTasksQueryVariables,
-    notifyOnNetworkStatusChange: true,
-    fetchPolicy: "network-only",
+    variables: queryVariables,
+    pollInterval,
   });
+  useNetworkStatus(startPolling, stopPolling);
 
-  useDisableTableSortersIfLoading(networkStatus);
-  const isOffline = useNetworkStatus();
-  const { showSkeleton } = usePollQuery({
-    networkStatus,
-    getQueryVariables,
-    refetch,
-    search,
-    isOffline,
-  });
+  useDisableTableSortersIfLoading(loading);
 
+  let showSkeleton = true;
+  if (data) {
+    showSkeleton = false;
```

Comment:
this means that the user will only see a loading skeleton when the table initially loads. I think this could lead to a smoother experience and we should do this for the other tables as well

Reply:
This was actually not intended and was a typo on my part. The behavior is the same with or without it.